skimage/filters/tests/test_thresholding.py

```diff
@@ -152,6 +164,29 @@ def test_otsu_astro_image():
     img = skimage.img_as_ubyte(data.astronaut())
     assert 109 < threshold_otsu(img) < 111
 
+def test_li_camera_image():
+    camera = skimage.img_as_ubyte(data.camera())
+    assert 63 < threshold_li(camera) < 65
+
+
+def test_li_coins_image():
+    coins = skimage.img_as_ubyte(data.coins())
+    assert 95 < threshold_li(coins) < 97
+
+
+def test_li_coins_image_as_float():
+    coins = skimage.img_as_float(data.coins())
+    assert 0.37 < threshold_li(coins) < 0.38
+
+
+def test_li_lena_image():
```

Comment:
Please remove the lena example, we're slowing removing lena from scikit-image (due to the provocative nature of the image).